### PR TITLE
feat: Added on Failure Function

### PR DESCRIPTION
**Pull request by uzirthapa**

### CODE REVIEW

File: lib/flutter_pw_validator.dart [hunk 1]
```diff
@@ -85,10 +87,19 @@ class _FlutterPwValidatorState extends State<FlutterPwValidator> {
     //Checks if all condition are true then call the user callback
     int conditionsCount = conditionsHelper.getter().length;
     int trueCondition = 0;
+    bool isFalse = false;
     for (bool value in conditionsHelper.getter().values) {
       if (value == true) trueCondition += 1;
+      if (value == false && conditionsCount != trueCondition) {
```
Comment (akirakakar) [inline lib/flutter_pw_validator.dart]: else if?

File: example/lib/main.dart [hunk 2]
```diff
@@ -29,51 +29,54 @@ class AppHome extends StatelessWidget {
...
+              flex: 7,
+              child: new Column(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: [
+                  Padding(
+                    padding: const EdgeInsets.symmetric(horizontal: 2.0),
+                    child: new TextField(
+                        controller: controller,
+                        decoration: new InputDecoration(
+                            hintText: "Password",
```
Comment (akirakakar) [inline example/lib/main.dart:48]: This should be string constant(translation constant) as well

File: example/lib/main.dart [hunk 3]
```diff
@@ -29,51 +29,54 @@ class AppHome extends StatelessWidget {
...
+                    minLength: 8,
+                    uppercaseCharCount: 2,
+                    numericCharCount: 3,
+                    specialCharCount: 1,
+                    width: 400,
+                    height: 150,
+                    onSuccess: () {
+                      print("Matched");
+                      Scaffold.of(context).showSnackBar(new SnackBar(
+                          content: new Text("Password is matched")));
```
Comment (akirakakar) [inline example/lib/main.dart:66]: Should be constant as well

**Review by akirakakar — APPROVED** [review]

File: example/lib/main.dart (hunk 2 above)
Comment (uzirthapa) [reply example/lib/main.dart:48]: This is just the example. so no worries there

File: example/lib/main.dart (hunk 3 above)
Comment (uzirthapa) [reply example/lib/main.dart:66]: Just an example.

File: lib/flutter_pw_validator.dart (hunk 1 above)
Comment (uzirthapa) [reply lib/flutter_pw_validator.dart]: Yeah this sounds good.

**Pull request merged**